examples/accounts/accounts.pact

```diff
@@ -141,11 +141,6 @@
               }
       )))
 
-  (defconst PACT_REF "ref")
```

Comment:
It would be nice if this continued to work...

Reply:
It's the change in Compile that forces this. We can't build an object with terms; in the case of `defconst`, these are eagerly evaluated -- but at module load, not compile. We would need actual `define`s like C to do this.